docs/content.zh/docs/deployment/resource-providers/standalone/kubernetes.md

```diff
@@ -184,18 +190,21 @@ flink-taskmanager-64847444ff-7rdl4   1/1     Running            3          3m28s
 flink-taskmanager-64847444ff-nnd6m   1/1     Running            3          3m28s
 ```
 
-You can now access the logs by running `kubectl logs flink-jobmanager-589967dcfc-m49xv`
+现在可以通过运行 `kubectl logs flink-jobmanager-589967dcfc-m49xv` 来访问日志。
```

Comment:
```suggestion
现在你可以通过运行 `kubectl logs flink-jobmanager-589967dcfc-m49xv` 来访问日志。
```
And would you mind supplementing the  subject of some statements in other parts of the page?